packages/server/providers.yaml

```diff
@@ -1,3 +1,13 @@
+adobe:
+    auth_mode: OAUTH2
+    authorization_url: https://ims-na1.adobelogin.com/ims/authorize/v2
```

Comment:
This looks like a datacenter-specific subdomain. Are you sure this is not something that needs to be passed by the developer, as [Connection Configuration](https://docs.nango.dev/reference/frontend-sdk#connection-config)?